### PR TITLE
acq stitching tiledacq: add registrar and weaver arguments

### PR DESCRIPTION
**Pull request by pieleric**
These argument allow to adjust the exact behaviour of the tiling.
In particular, in some case we want to tile without overlap (because the
stage precisition is good enough). In such case, the default registrar
would get confused/sad. It'd fallback to IDENTITY, but generate a lot of
warnings on the way.

### CODE REVIEW

**Review by lazem — APPROVED** [review]

**Comment by pieleric** [comment]
I hadn't properly tested the changes. This new version seems to really work, at least on the simulator. Please @lazem, have a second look.

**Comment by lazem** [comment]
> I hadn't properly tested the changes. This new version seems to really work, at least on the simulator. Please @lazem, have a second look.

Sure @pieleric I will have a deeper look.

File: src/odemis/acq/stitching/test/tiledacq_test.py [hunk 1]
```diff
@@ -244,6 +247,19 @@ def test_whole_procedure(self):
         self.assertIsInstance(data[0], model.DataArray)
         self.assertEqual(len(data[0].shape), 2)
 
+    def test_registrar_weaver(self):
+
+        overlap = 0.05  # Little overlap, no registration
+        area = (0, 0, 0.00001, 0.00001)
```
Comment (lazem) [inline src/odemis/acq/stitching/test/tiledacq_test.py:253]: This area is smaller than the fov, so it only stitches 1 image. Suggest to change it to:
```
sem_fov = compute_scanner_fov(self.ebeam)
area = (0, 0, sem_fov[0], sem_fov[1])
```

**Review by lazem — CHANGES_REQUESTED** [review]
Looks good, I suggest one small change to the test.

**Review by lazem — APPROVED** [review]

**Pull request merged**